src/systems/sprites/sprite.vert

```diff
@@ -0,0 +1,28 @@
+#version 300 es
+
+uniform mat4 modelViewMatrix;
+uniform mat4 projectionMatrix;
+
+in vec4 mvCol0;
+in vec4 mvCol1;
+in vec4 mvCol2;
+in vec4 mvCol3;
+in vec3 a_vertices;
+in vec2 a_uvs;
+out vec2 v_uvs;
+in float a_hubs_EnableSweepingEffect;
+flat out float v_hubs_EnableSweepingEffect;
```

Comment:
Kinda wish this effect was done in another forward rendered pass instead of having to be injected into each of our shaders, but that's out of the scope of this PR.

Reply:
I would like to learn more about how this can be done!